src/components/Organisms/ToDoCard/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import AddTaskButton from "../../Atoms/AddTaskButton/index";
+import Task from "../../Molecule/Task/index";
+import COLOR from "../../../variables/color";
+
+const TodoCard = () => {
+  const [taskList, setTaskList] = useState([]);
+
+  const onAddTaskButtonClick = () => {
+    setTaskList((taskList) => [...taskList, { name: "", initializing: true }]);
+  };
+
+  const onTaskComplete = (taskIndex) => {
+    const taskComplete = taskList.filter((task, index) => {
+      return index !== taskIndex;
+    });
+    setTaskList(taskComplete);

```

Comment:
```suggestion
    const filteredTaskList = taskList.filter((task, index) => {
      return index !== taskIndex;
    });
    setTaskList(filteredTaskList);
```
こうすると、フィルタリングされた `taskList` ということで、変数の中身を正確に表している命名になると思います